### ISSUE TITLE
maven repo for aar files?

### ISSUE THREAD

**Issue by z0lope0z** [spotify/android-sdk#109]
possibly on maven central? I'm currently hosting the files manually (not sure if this is legal/okay) but suggest this be done on github if for some reason you guys can't do it on maven central 
really makes my life much easier (as opposed to downloading and building it manually, yuck!)
like so:
```
repositories {
    maven {
        url "https://github.com/z0lope0z/raws/raw/master"
    }
    mavenCentral()
}
```

```
dependencies {
    compile 'com.spotify.sdk:spotify-player:1.0.0-beta9@aar'
    compile 'com.spotify.sdk:spotify-auth:1.0.0-beta9@aar'
}
```
If this isn't cool with you guys I'd be happy to bring the files down, just wondering why it isn't supported yet :3

**Comment by Wolftein** [comment]
+1 This will actually help a lot.

**Comment by z0lope0z** [comment]
Aaah, okay! But it should be fine releasing beta builds on mavenCentral though:
1) It's properly labelled as beta
2) The project will be able to test how well it does in releasing to mavencentral
3) Users can beta test the part wherein they download the sdk via mavencentral

I understand spotify probably wants to make a huge announcement once things are more stable, but the easier one can use the sdk, the more users. More users will lead to more testers which will lead to better stability overall.

In addition, I wouldn't recommend downloading the jar file I hosted on my github. I might have tampered with it and added malicious code. :3

**Comment by kaaes** [comment]
@z0lope0z I agree hosting the library on mavenCentral would be a great addition to the SDK and it is something that we have in our backlog. We will take it up and see if we can prioritise this feature.

I appreciate your effort and understand the reason behind it, but I am going to remove the link you posted from this thread, as it's not official or in any way endorsed by Spotify.

**Comment by emptyflash** [comment]
+1

**Comment by ryandt** [comment]
Thank you for hosting the `auth` library. Looking forward to a hosted version of the `player` library!

**Comment by n1ru4l** [comment]
@kaaes  Still no update?
